runner/src/main.rs

```diff
@@ -1379,14 +1379,41 @@ fn run_cargo_fmt(mode: FormatMode) -> Step {
     }
 }
 
-fn run_cargo_test() -> Step {
+fn run_cargo_test(cleanup: bool) -> Step {
     Step::Multiple {
         name: "cargo test".to_string(),
         steps: crate_manifest_files()
             .map(to_string)
-            .map(|entry| Step::Single {
-                name: entry.clone(),
-                command: Cmd::new("cargo", &["test", &format!("--manifest-path={}", &entry)]),
+            .map(|entry| {
+                if cleanup {
+                    Step::Multiple {
+                        name: entry.clone(),
+                        steps: vec![
+                            Step::Single {
+                                name: "run".to_string(),
+                                command: Cmd::new(
+                                    "cargo",
+                                    &["test", &format!("--manifest-path={}", &entry)],
+                                ),
+                            },
+                            Step::Single {
+                                name: "cleanup".to_string(),
+                                command: Cmd::new(
+                                    "rm",
+                                    &["-rf", &entry.replace("Cargo.toml", "target")],
```

Comment:
Is this going to delete the entire hard disk if entry contains an empty string or something like that? Seems a bit risky, maybe add some additional check just in case?

Reply:
Hopefully not :D 
Added some checks :)